### PR TITLE
Organization guards

### PR DESCRIPTION
**Pull request by ioaNikas**
- [x] OrganizationActiveGuard sets the active Organization.
- [x] OrganizationListGuard sets all the user Organizations.
- [x] Cleaned deprecated subscribes and store.set.

### CODE REVIEW

File: libs/organization/src/lib/organization.module.ts [hunk 1]
```diff
@@ -28,17 +29,18 @@ export const organizationRoutes: Routes = [
     pathMatch: 'full'
   },
   {
-    path: 'list',
-    component: OrgListComponent,
+    path: 'form',
+    component: OrgFormComponent
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization.module.ts]: This component should be removed at some point no ?

File: libs/organization/src/lib/guards/organization-list.guard.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    private afAuth: AngularFireAuth,
+    store: OrganizationStore,
+    router: Router
+  ) {
+    super(store, router);
+  }
+
+  get query() {
+    return this.afAuth.authState
+      .pipe(switchMap(user => this.fireQuery.fromQuery<Organization[]>(organizationQuery(user.uid))));
```
[the remark this answers is not in the document]
Comment (ioaNikas) [reply libs/organization/src/lib/guards/organization-list.guard.ts]: Are you ok with this instead to match what we will do in MovieListGuard ?

```typescript
return this.afAuth.authState.pipe(
      switchMap(user => {
        const query = organizationQuery(user.uid);
        return this.fireQuery.fromQuery(query);
      })
    );
```

File: libs/organization/src/lib/organization.module.ts (hunk 1 above)
Comment (ioaNikas) [reply libs/organization/src/lib/organization.module.ts]: For the moment, we only got the `Form` component to create an Organization. Unlike what we wrote on our Routes google.doc, there is no `Edit` component yet for Organizations.

File: libs/organization/src/lib/guards/organization-list.guard.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (GrandSchtroumpf) [reply libs/organization/src/lib/guards/organization-list.guard.ts]: If you want, but it's the same, with more lines, and not more readable...

**Pull request merged**